.github/workflows/bump.yml

```diff
@@ -20,15 +26,88 @@ jobs:
         id: bump
         with:
           majorVersion: ${{ inputs.majorVersion }}
-      
       - name: merge and write Changelogs
         id: merge-changelogs
         uses: sap/cloud-sdk-js/.github/actions/merge-and-write-changelogs@main
         env:
           VERSION: ${{ steps.bump.outputs.version }}
-
       - uses: sap/cloud-sdk-js/.github/actions/commit-and-tag@main
         name: Commit and tag
         with:
           version: ${{ steps.bump.outputs.version }}
           user-name: sap-ai-sdk
+
+  prepare-release-notes:
+    name: Prepare Release Notes
+    runs-on: ubuntu-latest
+    needs: [bump]
+    steps:
+      - name: Checkout current repository
+        uses: sap/ai-sdk-js/.github/actions/setup@main
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      - name: Checkout Docs Repo
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ env.DOCS_REPO }}
+          token: ${{ secrets.BOT_SDK_JS_FOR_DOCS_REPO_PR }}
+          path: ${{ env.DOCS_CHECKOUT_PATH }}
+          fetch-depth: 0
+      - name: Update release notes file
+        id: update-notes
+        run: |
+          NODE_OPTIONS="--loader ts-node/esm" pnpm ts-node --esm scripts/execute-add-changelog.ts
```

Comment:
Originally in cloud-sdk it was 
`npx ts-node -e "import { addCurrentChangelog } from './scripts/add-changelog'; addCurrentChangelog()"`

But, I faced issues with using -e and --esm, so switched to using a custom script to execute `addCurrentChangelog()` instead. Do suggest if you have better ways to handle this.

Reply:
Used `node` now instead.